src/main/java/org/minions/devfund/lourdes/Movie.java

```diff
@@ -0,0 +1,85 @@
+package org.minions.devfund.lourdes;
+
+import java.util.ArrayList;
+
+/**
+ * Class that define the Movie entity.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Constructor that receive the Movie Name.
+     *
+     * @param name represents the movie name.
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.actors = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)